backend/btrixcloud/operator.py

```diff
@@ -509,13 +533,13 @@ async def mark_finished(
         if redis:
             await self.add_crawl_errors_to_db(redis, crawl_id)
 
-        status.state = state
-        status.finished = to_k8s_date(finished)
-
-        if crawl:
-            await self.inc_crawl_complete_stats(crawl, finished)
-
-        return status
+        if state in SUCCESSFUL_STATES:
+            await add_successful_crawl_to_auto_add_collections(
+                self.crawls, self.crawl_configs, crawl_id, cid
+            )
+            await update_crawl_collections(self.collections, self.crawls, crawl_id)
+        else:
```

Comment:
This is probably no longer needed as we're only adding succesfull crawls, right?
Probably also should just have `add_successful_crawls_to_auto_add_collections` also call `update_crawl_collections`, so the surface in operator is just one function that handles collections?

Reply:
Good call! Updated